lib/resources/r4/encounter.yaml

```diff
@@ -528,6 +528,22 @@ fields:
       }
     }
 
+- name: location
+  required: 'No'
```

Comment:
Should required be 'Yes' since either serviceProvider or location must be provided?

Reply:
changed it to yes https://github.com/cerner/fhir.cerner.com/pull/803/commits/1d0f2634bd131127317a63f5458e2f27fc579829#diff-ea755c3df7ed8869a54a274cedf2e40e119b2f16a6e3ed8bc8ae211e0e87389cR536 the notes can do the job for the  mutual exclusive part.